### PR TITLE
Update and clean CSS code

### PR DESCRIPTION
**Pull request by Kingjosh007**
In this version, I: 

- Fixed a typo in the README file (will -> would)
- Wrapped my logo text in a link tag (a) 
- Updated my CSS to overwrite the default styling of <a> elements.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -167,10 +167,6 @@ ul li {
   margin: 12px;
 }
 
-image {
-  position: absolute;
-}
-
 .hr-image-container {
   display: flex;
```
Comment (Emmyn5600) [inline style.css:161]: - [ ] Nice to have logo on the left side however, logo has a link (a tag), please use a tag for the logo.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Kingjosh007 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

 Your project looks awesome however you went beyond the requirements, you would have created one branch to keep all the changes and merge it to the master branch after being approved instead of creating multiple branches as you did.
 
*  #### Please include the descriptive PR explaining what you did in the short summary, click on three dots to open the PR as well as create it.

 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css (hunk 1 above)
Comment (Kingjosh007) [reply style.css:161]: Thanks for the review. Please could you explain what you mean by "Please include the descriptive PR explaining what you did in the short summary, click on three dots to open the PR as well as create it." ?

Comment (Emmyn5600) [reply style.css:161]: it seems `No description provided`. however you are supposed to create it once you are requesting for a review, so click on the three dots to open it.

Comment (Kingjosh007) [reply style.css:161]: Okay, doing it right away.

Comment (Kingjosh007) [reply style.css:161]: I thought creating multiple branches (one per feature or enhancement) was actually a good practice, that's why I did. Sorry if I got it wrong. Would you please explain to me why I shouldn't do that?

Comment (Kingjosh007) [reply style.css:161]: Sorry, I'm confused. It seems like you're not reviewing the latest commit. I fixed this image style 2 commits earlier... 
I'm not sure what you're still waiting from me (given the latest commit).

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Kingjosh007** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### **To Highlight**
- The linters are passing.
- The commit are clear.

 ## Required Changes ♻️
_Check the comments under the review._

- The `main` tag should be `section` tag see the requirement 
sce



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @Kingjosh007, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**